### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by jasonacox** [jasonacox/tinytuya#425] [Announcements]
I'm grateful for this incredible community! Thanks to all of you who contribute, open issues, share experiences, help answer questions and make suggestions. You are what makes this project so fun!

A huge thanks to @uzlonewolf, co-maintainer, protocol guru, and key author of several brilliant contributions that have helped nudge TinyTuya forward. 🙏 

The best is yet to come... 

Cheers!

**Comment by mrioan** [comment]
Hello, I have some code I would like to contribute. What is the process? Shall I push a new branch and create a PR? How? I do not have access. Thanks

**Reply by jasonacox** [reply]
Thanks @mrioan for your interest in contributing. 🙏 

You can submit a PR by creating a fork of tinytuya in your own account, make your code changes and then submit a PR back to tinytuya for reivew.  A helpful guide is here: https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project

**Comment by Stephen2615** [comment]
I just wanted to say Thank You to @jasonacox and @uzlonewolf.  If all devs on the net had the same amount of enthusiasm with documentation, the world would be a far better place.

**Comment by jasonacox** [comment]
# And we have reached 1,000 Stars! ⭐ 

Congratulations everyone! Thanks again to all who contributed code, opened issues, made suggestions, provided help... or just clicked the star. 😁 🙏
